server/src/db/migrations/20211102194503_machines_table.ts

```diff
@@ -26,15 +26,30 @@ export async function up(knex: Knex): Promise<void> {
       }
     })
     .then(async () => {
-      const exists = await knex.schema.hasTable("Reservations");
+      const exists = await knex.schema.hasTable('Reservations');
       if (!exists) {
-        return knex.schema.createTable("Reservations", function (t) {
-          t.increments("id").primary();
-          t.integer("userId").references("id").inTable("Users");
-          t.integer("equipmentId").references("id").inTable("Equipment");
-          t.timestamp("createdAt").defaultTo(knex.fn.now());
-          t.time("startTime");
-          t.time("endTime");
+        return knex.schema.createTable('Reservations', function (t) {
+          t.increments('id');
+          t.integer('creator').references('id').inTable('Users');
+          t.integer('maker').references('id').inTable('Users');
+          t.integer('labbie').references('id').inTable('Users');
+          t.timestamp('createDate').defaultTo(knex.fn.now());
+          t.time('startTime');
+          t.time('endTime');
+          t.integer('equipment').references('id').inTable('Equipment');
+          t.enu('status', ['PENDING', 'CONFIRMED', 'CANCELLED']);
+          t.timestamp('lastUpdated').defaultTo(knex.fn.now());
+        });
+      }
+    }).then(async () => {
+      const exists = await knex.schema.hasTable('ReservationEvents');
+      if (!exists) {
+        return knex.schema.createTable('ReservationEvents', function (t) {
```

Comment:
I feel like there should be a foreign key on this table to reference the reservation the event is for

Reply:
@pjw9963 right, I didn't mean to include this Reservation stuff, I've mixed up some of the stuff from my other PR. But fixing now that I have